client/src/components/Search/Search.jsx

```diff
@@ -11,14 +11,22 @@ import "./Search.css";
 // useContext
 import { BusinessContext } from "../useContext/BusinessContext";
 import {useHistory} from 'react-router-dom';
+import axios from "axios";
 import logo from "../../images/logo.png";
 
 export default function Search() {
   const [searchValue, setSearchValue] = useState("");
   const [location, setLocation] = useState("");
   // may or may not need coordinates
   const [coordinates, setCoordinates] = useState({ lat: null, lng: null });
+  //select date 
   const [selectedDate, setSelectedDate] = useState("");
+  //select start time 
+  const [selectedStartTime, setSelectedStartTime] = useState("");
+  //select end time 
+  const [selectedEndTime, setSelectedEndTime] = useState("");
```

Comment:
your comments here are unnecessary because we can just read the variable names....comments are only needed if it's unclear at quick glance what the code is/does.

Reply:
Thx, I did it!